Formula/gost-proxy.rb

```diff
@@ -0,0 +1,48 @@
+class GostProxy < Formula
+  desc "GO Simple Tunnel - a simple tunnel written in golang"
+  homepage "https://github.com/ginuerzh/gost"
+  url "https://github.com/ginuerzh/gost/archive/v2.6.1.tar.gz"
+  sha256 "d8231f5869e54066e0f1ba333d0209727bfe14998d296ebdf7b423d0c33209c6"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    mkdir "src/github.com/ginuerzh/gost"
```

Comment:
Please take a look at https://github.com/Homebrew/homebrew-core/blob/fe96795085a6e357a7aa0b5a2764ad0653bc0faa/Formula/annie.rb#L17-L22 as a good example of how to build go formula.